stream_alert/rule_processor/main.py

```diff
@@ -18,25 +18,27 @@
 import os
 
 from stream_alert.rule_processor.handler import StreamAlert
-from rules import (
-    sample_matchers
-)
 
 modules_to_import = set()
 # walk the rules directory to dymanically import
-for root, dirs, files in os.walk('rules/'):
-    # ignore old rule files and helpers
-    if root in ['rules/helpers', 'rules/']:
-        continue
-    # ignore __init__.py files
-    filtered_files = filter(lambda x: not x.startswith('.') and
-                                      not x.endswith('.pyc') and
-                                      not x.startswith('__init__'), files)
-    for import_file in filtered_files:
-        package_path = root.replace('/', '.')
-        import_module = os.path.splitext(import_file)[0]
-        modules_to_import.add('{}.{}'.format(package_path, import_module))
-
+for folder in ('matchers/', 'rules/'):
+    for root, dirs, files in os.walk(folder):
+        filtered_files = filter(lambda x: not x.startswith('.') and
```

Comment:
That's fair, I'll convert it (even though the ruby programmer in me really loves filter and map)